### ISSUE TITLE
Support for Next.js

### ISSUE THREAD

**Issue by lucassaid** [polemius/recoil-persist#13]
Hello! I'm trying to make this work with Next.js, but unfortunately it [doesn't support ES modules yet](https://github.com/vercel/next.js/issues/9607), so I'm getting this error:
```
import { useTransactionObservation_UNSTABLE } from 'recoil'
^^^^^^

SyntaxError: Cannot use import statement outside a module
```

I would love to help you adding support for CommonJS. I'm actually trying to make it work following [this guide](https://2ality.com/2019/10/hybrid-npm-packages.html) (option 4) but with no success so far. Can you tell me if I'm in the good path?

Thanks a lot for your attention and for this package!

**Comment by polemius** [comment]
I have used [this library](https://github.com/ai/dual-publish/) to publish CommonJS + ES modules library. I have published it as 1.0.0 release.

**Comment by lucassaid** [comment]
Worked like a charm. Thanks!
